src/SequenceManager.cc

```diff
@@ -300,7 +332,7 @@ void SequenceManager::rename_sequence(const SequenceName& old_name, UniqueId uni
 void SequenceManager::rename_sequence(Sequence& sequence, const SequenceName& new_name)
```

Comment:
Maybe you can explain why you set the new name in combination with the other `rename_sequence` member function. I am little but confused.